src/main/java/com/google/sps/data/Book.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.lang.String;
+
```

Comment:
Anand recommended that we added javadocs to these. This is what I followed to do that: https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
Oh ya, forgot about that. Thanks!